test/command_line/test_find_spots_server_client.py

```diff
@@ -14,6 +15,9 @@ def start_server(server_command, working_directory):
 
 
 def test_find_spots_server_client(dials_data, tmpdir):
+    if sys.hexversion >= 0x3080000 and sys.platform == "darwin":
```

Comment:
Makes me wonder how many places we look at the version in a less reliable manner 🙄